pkg/agent/manager/manager.go

```diff
@@ -285,6 +285,9 @@ func (m *manager) runSynchronizer(ctx context.Context) error {
 
 		err := m.synchronize(ctx)
 		switch {
+		case nodeutil.IsUnknownAuthorityError(err):
+			m.c.Log.WithError(err).Info("Synchronize failed, non-recoverable error")
+			return fmt.Errorf("agent SVID is signed by unknow authority: %w", err)
```

Comment:
Agent SVID or Server SVID?

I think the error message we print with `%w` is going to be roughly the same as this one? Does it include any add'l information? Maybe the string should be something like `failed to sync with SPIRE Server: %w`?